gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/pages/PermissionsPage.java

```diff
@@ -58,18 +59,21 @@ public PermissionsPage() {
 		disableLink(this.permissionsPageLink);
 	}
 
-	public PermissionsPage(final GbUser taSelected) {
-		disableLink(this.permissionsPageLink);
-		this.taSelected = taSelected;
-	}
-
 	@Override
 	public void onInitialize() {
 		super.onInitialize();
 
+		// grab the selected parameter (if provided)
+		final PageParameters params = getPageParameters();
+		final Optional<String> taUuid = Optional.ofNullable(params.get("selected").toOptionalString());
```

Comment:
Yeah, it doesn't need to be an Optional, I only made it one to match up with `taSelected` which is now an Optional... it also felt like a candidate, as it helps ensure we avoid a NPE.  Happy to refactor?